R/init.R

```diff
@@ -111,41 +111,41 @@
 #' @include modules.R
 #'
 #' @examples
```

Comment:
Hmmm still thinking about the example for this `init()` function...
To me, most of the people looking at this example would came from clinical trials domain so it would be good to speak in the same language - i.e. use ADAM model. WDYT @lcd2yyz ?

Reply:
I can copy paste data created in `get_dummy_data` so we can have an ADaM alike data here.

Reply:
There is also vignette `including-adam-data-in-teal.Rmd` for people who want to use teal in clinical trials domain. I'd prefer to keep `teal::init` documentation generic.